project/src/utils/utils.ts

```diff
@@ -1,9 +1,18 @@
 import { Offer, OfferFromServer } from '../types/offer';
 import { Review, ReviewFromServer } from '../types/review';
+import dayjs from 'dayjs';
 
 const ONE_STAR_RATING_PERCENT = 20;
 
-const getRatingInStars = (rating: number): string => `${ONE_STAR_RATING_PERCENT*rating}%`;
+const getRatingInStars = (rating: number): string => `${ONE_STAR_RATING_PERCENT*Math.round(rating)}%`;
+
+const sortDateDown = (cardA : Review, cardB : Review) : number => dayjs(cardB.date).diff(dayjs(cardA.date));
+
+const disableNewCommentForm = () :void => document.querySelectorAll('form input, form textarea, form button').forEach((elem) => elem.setAttribute('disabled', 'disabled'));
```

Comment:
```suggestion
const disableNewCommentForm = (isDisabled) :void => document.querySelectorAll('form input, form textarea, form button').forEach((elem) => elem.disable = isDisabled;
```
тут можно одну функцию. которая это делает

но лучше всего, вообще убрать все эти функции, и сделать дизейбл прям в компоненте где это надо